generate/spago.yaml

```diff
@@ -1,9 +1,42 @@
 workspace:
   lock: true
   package_set:
-    registry: 24.4.0
+    registry: 47.9.1
   extra_packages:
     registry-lib:
       git: https://github.com/purescript/registry-dev.git
       ref: master
       subdir: lib
+    language-cst-parser:
+      git: https://github.com/natefaubion/purescript-language-cst-parser.git
+      ref: "v0.13.0"
+      dependencies:
+        - "arrays"
+        - "console"
+        - "const"
+        - "control"
+        - "effect"
+        - "either"
+        - "enums"
+        - "foldable-traversable"
+        - "free"
+        - "functions"
+        - "functors"
+        - "identity"
+        - "integers"
+        - "lazy"
+        - "lists"
+        - "maybe"
+        - "newtype"
+        - "node-process"
+        - "numbers"
+        - "ordered-collections"
+        - "partial"
+        - "prelude"
+        - "st"
+        - "strings"
+        - "transformers"
+        - "tuples"
+        - "typelevel-prelude"
+        - "unfoldable"
+        - "unsafe-coerce"
```

Comment:
```suggestion
        - "unsafe-coerce"

```

Reply:
applied in 4b81bcc268a80ff94b946df46b8169e782227d52